src/odemis/driver/orsay.py

```diff
@@ -1389,3 +1440,1247 @@ def terminate(self):
+                                                                   self._ionColumn.ObjectiveShiftY_Maxvalue])
+        self._horizontalFOVConnector = OrsayParameterConnector(self.horizontalFOV, self._ionColumn.ObjectiveFieldSize,
+                                                               minpar=self._ionColumn.ObjectiveFieldSize_Minvalue,
+                                                               maxpar=self._ionColumn.ObjectiveFieldSize_Maxvalue)
+        self._measuringCurrentConnector = OrsayParameterConnector(self.measuringCurrent, self._ionColumn.FaradayStart,
+                                                                  conversion={True: 1, False: 0})
+        self._currentConnector = OrsayParameterConnector(self.current, self._ionColumn.FaradayCurrent,
+                                                         minpar=self._ionColumn.FaradayCurrent_Minvalue,
+                                                         maxpar=self._ionColumn.FaradayCurrent_Maxvalue)
+        self._videoDelayConnector = OrsayParameterConnector(self.videoDelay, self._ionColumn.VideoDelay)
+        self._flybackTimeConnector = OrsayParameterConnector(self.flybackTime, self._ionColumn.FlybackTime)
+        self._blankingDelayConnector = OrsayParameterConnector(self.blankingDelay, self._ionColumn.BlankingDelay)
+        self._rotationConnector = OrsayParameterConnector(self.rotation, self._ionColumn.ObjectiveScanAngle)
+        self._dwellTimeConnector = OrsayParameterConnector(self.dwellTime, self._ionColumn.PixelTime,
+                                                           minpar=self._ionColumn.PixelTime_Minvalue,
+                                                           maxpar=self._ionColumn.PixelTime_Maxvalue)
+        self._contrastConnector = OrsayParameterConnector(self.contrast, self._sed.PMT, factor=0.01)
+        self._brightnessConnector = OrsayParameterConnector(self.brightness, self._sed.Level, factor=0.01)
+        self._operatingModeConnector = OrsayParameterConnector(self.operatingMode,
+                                                               self._datamodel.Scanner.OperatingMode,
+                                                               conversion={True: 1, False: 0})
+        # Subscribe to the parameter on the Orsay server
+        self._ionColumn.ImageSize.Subscribe(self._updateImageFormat)
+        self._ionColumn.ImageArea.Subscribe(self._updateTranslationResolution)
+
+        self._connectorList = [x for (x, _) in  # save only the names of the returned members
+                               inspect.getmembers(self,  # get all members of this FIB_source object
+                                                  lambda obj: type(obj) == OrsayParameterConnector  # get only the
+                                                  # OrsayParameterConnectors from all members of this FIB_source object
+                                                  )
+                               ]
```

Comment:
Why don't you call update_VAs() from here, as it's done in the other components? Do the connectors already take care of setting the VA's value at init? It'd still require to update the translation, resolution VAs, right?

Reply:
Because I forgot... The connectors do indeed take care of it at initialisation, but the call is indeed needed for imageFormat, translation and resolution. I've changed it in all three FIB devices.